fv3core/stencils/remap_profile.py

```diff
@@ -509,55 +509,89 @@ class RemapProfile:
     This corresponds to the cs_profile routine in FV3
     """
 
-    def __init__(self, kord, iv):
+    def __init__(
+        self,
+        kord: int,
+        iv: int,
+        i1: int,
+        i2: int,
+        j1: int,
+        j2: int,
```

Comment:
Shoot, I completely forgot you'd have to propagate this to the RemapProfile class! Sorry to ask for so much extra work!